### PR TITLE
Add command to get the hash of the git commit that is currently being run

### PR DESCRIPTION
**Pull request by LinuxMercedes**

### CODE REVIEW

**Comment by streed** [comment]
:hankey:

File: modules/version.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var exec = require('child_process').exec;
+
+var hash;
+
+module.exports.init = function(b) {
+	exec('git rev-parse HEAD', function(error, stdout, stderr) {
```
Comment (euank) [inline modules/version.js]: Please also add something like `require('../package/json').version` into the output.

File: modules/version.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		hash = stdout;
+		console.log(hash);
+	});
+}
+
+module.exports.run = function(r, parts, reply) {
+	reply(hash);
+}
+
+module.exports.commands = ['version', 'head', 'hash'];
```
Comment (euank) [inline modules/version.js]: Not a fan of having this many commands for it. Pick one, preferably with a long and inconvenient name :)

File: modules/version.js (hunk 1 above)
Comment (LinuxMercedes) [inline modules/version.js]: `child_process` is part of the npm stdlib; do I really need a package.json for that?

File: modules/version.js (hunk 2 above)
Comment (LinuxMercedes) [inline modules/version.js]: Aww, ok, fine.

File: modules/version.js (hunk 1 above)
Comment (streed) [inline modules/version.js]: The node package version hasn't changed, could just hardcode it and stay in beta forever.

Comment (LinuxMercedes) [inline modules/version.js]: Oh wow I'm dumb. Fixing.

**Pull request merged**